config/config_test.go

```diff
@@ -151,7 +141,7 @@ func TestMergeFlagsWithConfig(t *testing.T) {
 	for i, tt := range tests {
 		t.Run(fmt.Sprintf("testcase-%d", i+1), func(t *testing.T) {
 			c := DefaultConfig()
-			if err := mergeFlagsWithConfig(tt.flags, c); (err != nil) != tt.wantErr {
+			if err := mergeFlagsWithConfig(&tt.flags, c); (err != nil) != tt.wantErr {
```

Comment:
Could you simplify `(err != nil) != tt.wantErr` by moving `err != nil` to a variable?

Reply:
https://github.com/hazelcast/hazelcast-commandline-client/pull/61/commits/38d55d5d21f54ae0507644605f7f2c3a0fb6497c